react_ujs/src/renderComponent/withHotReload.js

```diff
@@ -0,0 +1,36 @@
+var ReactDOM = require("react-dom")
+var reactHotLoader = require("react-hot-loader")
+var AppContainer = reactHotLoader.AppContainer;
+
+// Render React component with hot reload.
+//
+// See the HMR section in README to ensure required steps are completed.
```

Comment:
Same for this one. Let me know if this comment could be more helpful, i.e. should the steps to setup be inlined here?